lib/code-caller/code-caller-native.js

```diff
@@ -280,6 +283,12 @@ class CodeCallerNative {
       this._startChild();
     }
 
+    // Give the child a reasonably long amount of time to start up. Under
+    // periods of high load, it should be cheaper to wait for a child to
+    // start slowly than it would be to time out too quickly and try to
+    // launch a new one to replace it.
+    await this.call('ping', null, null, 'ping', [], { timeout: 30 * 1000 });
```

Comment:
Could we shift this up to be inside the `this.state === CREATED` block, just after `_startChild()`?

Reply:
Done in 79d4ed1.